gems/mail/2.8/mail-generated.rbs

```diff
@@ -0,0 +1,5743 @@
+# encoding: utf-8
```

Comment:
Could you tell me why you named this file with `-generated` suffix?
I think it depends on the design whether this file should have `-generated` suffix or not. So I'd like to hear your intention.


If you designed this file as a "generated file", the suffix is necessary.
In this case, you also need to commit the generator script to update this file by other contributors.

If you designed this file as an editable file by humans, the suffix is unnecessary.
This file is actually generated, but it will be modified by humans in this case. This suffix is not meaningful for an editable file I think.

Reply:
> Could you tell me why you named this file with -generated suffix?

Because these rbs automatically generated by the `rbs prototype rb` command. I had assumed that the rbs automatically generated by the generator were customarily `-generated` suffix.

However, I think you are right that the suffix will not make sense as we update type definitions in the future. There seems to be little merit in having separate files for automatically generated rbs and rbs that should be edited by humans.

So I would like to remove the suffix from this file.